### PR TITLE
close registration

### PR DESCRIPTION
**Pull request by Aholicknight**
Since compcamps 2018 is done, this should be closed.

### CODE REVIEW

**Review by BrettMayson — APPROVED** [review]

**Comment by Aholicknight** [comment]
Is everything good to merge?

File: www/register.php [hunk 1]
```diff
@@ -7,11 +7,13 @@
...
 <section class="register-section">
-  <h2>Registration is Open</h2>
+  <h2>Registration is <b>CLOSED</b></h2>
   <h3>$350</h3>
   <h3>First week: <b>CLOSED</b></h3>
-  <h3>Second week: OPEN</h3>
-  <a class="button" href="dash/register">Register</a>
+  <h3>Secound week: <b>CLOSED</b></h3>
+  <form method="get" action="dash/register">
+  <button type="button" disabled>Register</button>
```
Comment (trollmad3) [inline www/register.php]: When compcamps starts in 2019 again you can remove `disabled` in the `<button>` tag.

File: www/register.php [hunk 2]
```diff
@@ -7,11 +7,13 @@
...
 <section class="register-section">
-  <h2>Registration is Open</h2>
+  <h2>Registration is <b>CLOSED</b></h2>
   <h3>$350</h3>
   <h3>First week: <b>CLOSED</b></h3>
-  <h3>Second week: OPEN</h3>
-  <a class="button" href="dash/register">Register</a>
+  <h3>Second week: <b>CLOSED</b></h3>
+  <form method="get" action="dash/register">
+  <button type="button" disabled>Register</button>
```
Comment (Aholicknight) [inline www/register.php:15]: When compcamps starts in 2019 again you can remove ` disabled` in the `<button>` tag.

**Comment by Aholicknight** [comment]
image
I noticed you merged it @synixebrett but its still **OPEN** on the website.

**Comment by BrettMayson** [comment]
It still needs to be pushed to live, and the deploy to the testing server didn't work. So I need to fix some SSH stuff first.

**Comment by Aholicknight** [comment]
Oh ok.

**Comment by Aholicknight** [comment]
Did you fix the SSH problem @synixebrett ?

**Pull request merged**